packages/forms-web-app/src/controllers/examination/session/submission-items-session.js

```diff
@@ -62,44 +74,60 @@ const getSubmissionItemComment = (submissionItem) => {
 	return submissionItem.comment;
 };
 
-const getActiveSubmissionItemKey = (session) => {
+const getActiveSubmissionItemId = (session) => {
 	const examinationSession = getExaminationSession(session);
-	const activeDeadlineItem = examinationSession.activeItem;
-	if (!activeDeadlineItem) return false;
-	return activeDeadlineItem;
+	return examinationSession.activeSubmissionItemId;
```

Comment:
these can be internal const ands used within the file to remove duplication. 

maybe the const can be the exact key to keep it simple ?

I am about to do this on another branch. 
return examinationSession[activeSubmissionItemId]